pkg/envoy/cell.go

```diff
@@ -103,6 +104,7 @@ func (r ProxyConfig) Flags(flags *pflag.FlagSet) {
 	flags.Bool("use-full-tls-context", false, "If enabled, persist ca.crt keys into the Envoy config even in a terminatingTLS block on an L7 Cilium Policy. This is to enable compatibility with previously buggy behaviour. This flag is deprecated and will be removed in a future release.")
 	flags.Uint32("proxy-xff-num-trusted-hops-ingress", 0, "Number of trusted hops regarding the x-forwarded-for and related HTTP headers for the ingress L7 policy enforcement Envoy listeners.")
 	flags.Uint32("proxy-xff-num-trusted-hops-egress", 0, "Number of trusted hops regarding the x-forwarded-for and related HTTP headers for the egress L7 policy enforcement Envoy listeners.")
+	flags.Duration("policy-restore-timeout", 3*time.Minute, "Maxiumum time to wait for enpoint policy restoration before starting serving resources to Envoy")
```

Comment:
what about prefixing this flag with `envoy-` (IMO better than `proxy-`) to provide some context? (yes, many older properties that we migrated lack this prefix - but IMO we should add it)

Reply:
The reason I did not do this is that the `envoy-` and `proxy-` typically translate to Envoy configuration, either on the command line, bootstrap, or filter configuration, whereas this is only about agent behavior that does not translate to any of these.

Reply:
ok, thanks for the clarification. well then it could also be an option to give some more context from another angle.

-> `policy-l7-restore-timeout`, `policy-envoy-restore-timeout` - but i leave this up to you. IMO it sounds too generic - especially since the property isn't used outside of envoy/proxy scope.